city_furniture/tests/test_furniture_signpost_api.py

```diff
@@ -61,6 +62,7 @@ def test__furniture_signpost_real__list(geo_format):
     assert response_data["count"] == 3
     for result in response_data["results"]:
         obj = FurnitureSignpostReal.objects.get(pk=result["id"])
+        assert result["device_type_description"] == DEFAULT_DEVICE_TYPE_DESCRIPTION
```

Comment:
Perhaps it would be good to add this same check to the detail endpoint test `test__furniture_signpost_real__detail`.

Reply:
Done